source/indices.d.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Generates a tuple based on the length of the input array
+ * and fills each index with the index number.
+ *
+ * @example type FruitIndices = Indices<['apple', 'banana', 'plum']> // [0, 1, 2]
+ * @example type FruitIndex = Indices<['apple', 'banana', 'plum']>[number] // 0 | 1 | 2
+ *
+ * @category Array
+ * @category Utility
+ */
+export type Indices<
```

Comment:
Check if it's possible to use `unknown` for any of the places you use `any`. I prefer to use `unknown` whenever possible.

Reply:
Works without issue 👍